pyuvdata/uvdata.py

```diff
@@ -2159,28 +2159,48 @@ def _key2inds(self, key):
         Returns:
             blt_ind1: numpy array with blt indices for antenna pair.
             blt_ind2: numpy array with blt indices for conjugate antenna pair.
-            pol_ind: numpy array with polarization indices
+                      Note if a cross-pol baseline is requested, the polarization will
+                      also be reversed. e.g. (1, 2, 'xy') -> conj(2, 1, 'yx')
```

Comment:
I find this comment confusing. I'm not sure when I'm getting reversed polarizations and whether that means they match what I asked for or not.